wab/src/main/java/org/apache/unomi/web/ContextServlet.java

```diff
@@ -136,11 +137,17 @@ public void service(ServletRequest request, ServletResponse response) throws Ser
             try {
                 contextRequest = mapper.readValue(factory.createParser(stringPayload), ContextRequest.class);
             } catch (Exception e) {
+                ((HttpServletResponse)response).sendError(HttpServletResponse.SC_BAD_REQUEST, "Check logs for more details");
                 logger.error("Cannot read payload " + stringPayload, e);
                 return;
             }
             scope = contextRequest.getSource().getScope();
         }
+        if (httpMethod.equals("POST") && StringUtils.isBlank(stringPayload)){
+            ((HttpServletResponse)response).sendError(HttpServletResponse.SC_BAD_REQUEST, "Check logs for more details");
+            logger.error("Invalid Resource, please check the request");
+            return;
+        }
```

Comment:
Could you explain the point of this code ? It was not, before modification, mandatory to have a payload and this modification makes it required. I don't think this is what is needed. All that we need is the modification on the error handling for the JSON parsing.